packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx

```diff
@@ -34,17 +34,21 @@ export type NavigationTabGroupTypes =
   (typeof NAVIGATION_TAB_GROUP_TYPES)[keyof typeof NAVIGATION_TAB_GROUP_TYPES];
 
 type TabItem = {
+  id: string;
   text: string;
   icon?: FunctionComponent<any> | null;
   href?: string;
+  dataCy?: string;
```

Comment:
Could you change the name from `cy` to something more descriptive as this is slightly confusing as to what cy means?

Reply:
Change dataCy to dataCypress as it used for cypress test .[ec83984](https://github.com/Skyscanner/backpack/pull/3607/commits/ec83984a5d56b0ad45bdf755a5cfc38615aecf4d)

Reply:
Thanks for your help. I have resolved this props issue .Updated it in [7b6a1d9](https://github.com/Skyscanner/backpack/pull/3607/commits/7b6a1d94188ebf04c662a5c5b43524a9835c602a)